$OpenBSD: patch-configure,v 1.3 2015/07/08 16:04:14 sthen Exp $
XXX hack for vax, remove when it gets shared libraries
--- configure.orig	Fri Jul  3 22:01:58 2015
+++ configure	Wed Jul  8 16:32:51 2015
@@ -62646,7 +62646,9 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_ext
     void __attribute__ ((constructor))
     stdbuf_init (void)
     {
+#if !__vax__
       stdbuf = 1;
+#endif
     }
 int
 main ()
